### ISSUE TITLE
I can't also implement unidac TUniConnection

### ISSUE THREAD

**Issue by relativ** [remobjects/pascalscript#170]
[SQLConnection.zip](https://github.com/remobjects/pascalscript/files/1723993/SQLConnection.zip)
Hello,

This time I tryed to implement TUniConnection in UniDAC component packages but I get same error with DB Express integration. when I activate connection with using Connected := true; property then it exit by code. even exit to application thread with no error.

Can you please check it where is wrong with this implementation.

thanks

**Comment by ronaldhoek** [comment]
Dit you already try something like:

```pascal
SQLConnection:= TSQLConnection.Create();
try
SQLConnection.ProviderName := 'MySQL'; // Access, Advantage, ASE, DB2, DBF, InterBase, MySQL, NexusDB, ODBC, Oracle, PostgreSQL, SQL Server, SQLite, MongoDB
SQLConnection.UserName := 'root';
SQLConnection.Password := 'toor';
SQLConnection.Server := 'localhost';
SQLConnection.Database := 'mysql';
SQLConnection.Open();
except
  MsgBox(GetExceptionMessage), mbCriticalError, mb_Ok);
end;
```
This way you can at least see the actual exception.
I don't use pascalscript on it's own (only with Inno Setup), so I'm not sure wether this will compile/run in your testcase....

**Comment by relativ** [comment]
Yes I did test between try except but it not return error back just show blank page :( 

I am trying to create web application platform using with pascalscript and now I need to implement some database component to bring database accesibility to the project.
Can you check my freetime project may you contribute on it 
https://github.com/relativ/pascalweb
thanks

**Comment by ronaldhoek** [comment]
I don't know how Exception Handling works in PascalScript. 
Maybe it's best to look into that first, zo you can capture exceptions.
